### ISSUE TITLE
Autocomplete == false not working

### ISSUE THREAD

**Issue by NicoBijl** [danielfarrell/bootstrap-combobox#227]
Hi , 

in the html template, please change autocomplete="false" to autocomplete="off".
http://www.w3schools.com/tags/att_input_autocomplete.asp

Thanks

**Comment by iamprofchaos** [comment]
Agreed! Thanks NicoBijl modified the js and this fixed the issue of the original selects dropdown showing over the one thats created

**Comment by falkorhein** [comment]
Thanks guys. I had the same issue. Modified js helps for the moment.

**Comment by weisborg** [comment]
I was just getting on here to write this very thing.  For some reason they decided autocomplete would be on/off.  To the other comment saying "original selects dropdown", it isn't the original select showing, it is the dropdown for autocomplete showing.
